nginx/Dockerfile

```diff
@@ -1,2 +1,8 @@
 FROM nginx:1.24
+ARG LEGOCHECKSUM="f5a978397802a2eb20771925ceb173dff88705b45fdbb2e68312269e205fa85d"
```

Comment:
Can we verify that it's fine that this is being committed to this, a public repository?

Reply:
Yes, this is publicly available.